GrandCentralBoard/Widgets/Slack/View/MessageBubbleView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MessageBubbleView.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 25.05.2016.
+//  Copyright © 2016 Macoscope. All rights reserved.
+//
+
+import UIKit
+
+
+@IBDesignable
+final class MessageBubbleView: UIView {
+
+    private lazy var imageView: UIImageView = { [unowned self] in
+        let imageView = UIImageView(image: UIImage(named: "message_bubble")!)
+        self.addSubview(imageView)
+        return imageView
+    }()
+
+    private lazy var label: UILabel = { [unowned self] in
+        let label = UILabel()
+        label.font = UIFont.systemFontOfSize(40)
+        label.textColor = UIColor.whiteColor()
+        label.numberOfLines = 0
+
+        self.addSubview(label)
+        return label
+    }()
+
+    @IBInspectable var text: String = "" {
+        didSet {
+            label.text = text
+            setNeedsLayout()
+        }
+    }
+
+    override func layoutSubviews() {
```

Comment:
That's a good point. I just tried to do it with autolayout, but I got an issue with the label having some extra margins. Just some extra pixels below the bottom line and border, and the top line and border. I don't see how I could change it, so I would like to stick with manual layout for extra control. Here is the code I used, in case you can point me in a better direction:

https://gist.github.com/michallaskowski/d92c90b2284200e095ab7d31391a3fba

<img width="280" alt="screen shot 2016-05-31 at 13 30 46" src="https://cloud.githubusercontent.com/assets/3706309/15672946/f4f64352-2733-11e6-9efd-b7a229b587cb.png">